sourcegraph/cody
# Add custom abort controller to circumvent max event listeners limit

Right now, the dev console in VS Code is throwing a bunch of warnings because we append more than 10 event listeners to the abort controller. Since this is only possible to change with Node APIs (which we won't have in none Node environments), this PR adds a lightweight custom abort controller implementation. 

## Test plan

- Enable graph context
- Start VS Code and click inside a few document section
- Observe that no more warning is thrown
